File: CombatTracker/CombatTracker.Web/ClientApp/app/entities/reference/SpeedChart.ts


module entities.reference {

    // $Classes/Enums/Interfaces(filter)[template][separator]
    // filter (optional): Matches the name or full name of the current item. * = match any, wrap in [] to match attributes or prefix with : to match interfaces or base classes.
    // template: The template to repeat for each matched item
    // separator (optional): A separator template that is placed between all templates e.g. $Properties[public $name: $Type][, ]

    // More info: http://frhagn.github.io/Typewriter/

    
    export class SpeedChart {
        
        // MSBASEDB
        public msBaseDB: number = 0;
        // MSFLEEEVADE
        public msFleeEvade: number = 0;
        // AQCHARGELUNGE
        public aqChargeLunge: number = 0;
        // AQINITIATIVE
        public aqInitiative: number = 0;
    }
}

File: CombatTracker/CombatTracker.Entities/Reference/Creatures/Charts/SpeedChart.cs
using CombatTracker.Entities.Reference.Base;

namespace CombatTracker.Entities.Reference.Creatures.Charts
{
    public partial class SpeedChart : ReferenceEntry
    {
        public SpeedChart()
        {
            
        }
    
        public int MSBaseDB { get; set; }
        public int MSFleeEvade { get; set; }
        public int AQChargeLunge { get; set; }
        public int AQInitiative { get; set; }
    
        public override string ToString()
        {
            return Name;
        }

    }
    
}
